src/main/resources/assets/minecolonies/lang/en_US.lang

```diff
@@ -220,3 +220,118 @@ com.minecolonies.coremod.gui.workerHuts.Deliveryman=Deliveryman
+achievement.fisher.fishCaught.25=Enough for my family.
+achievement.fisher.fishCaught.100=Plenty o' fish!
+achievement.fisher.fishCaught.500=Should I sell it?
+achievement.fisher.fishCaught.1000=Tale of a whale.
+achievement.guard.mobsKilled.one=Gotcha!
+achievement.guard.mobsKilled.25=Groovy.
+achievement.guard.mobsKilled.100=Thigh bone.
+achievement.guard.mobsKilled.500=Invasion.
+achievement.guard.mobsKilled.1000=I'm batman.
+achievement.miner.oresMined.one=What's that?
+achievement.miner.oresMined.25=Is that iron?
+achievement.miner.oresMined.100=Gold fever.
+achievement.miner.oresMined.500=It's conducting electricity?
+achievement.miner.oresMined.1000=Damn it! Lapis again...
+achievement.miner.diamondsMined.one=So shiney!
+achievement.miner.diamondsMined.25=Enough to make an armour!
+achievement.miner.diamondsMined.100=Diamonds are a girls best friend.
+achievement.miner.diamondsMined.500=Shine bright like a diamond!
+achievement.miner.diamondsMined.1000=I'm rich!
+achievement.lumberjack.treesFelled.one=Firewood.
+achievement.lumberjack.treesFelled.25=Logs.
+achievement.lumberjack.treesFelled.100=Enough for a house.
+achievement.lumberjack.treesFelled.500=Chainsaw master.
+achievement.lumberjack.treesFelled.1000=The forest.
+achievement.lumberjack.saplingsPlanted.one=My first sapling!
+achievement.lumberjack.saplingsPlanted.25=I could make a living out of this?
+achievement.lumberjack.saplingsPlanted.100=So many trees!
+achievement.lumberjack.saplingsPlanted.500=A Forester.
+achievement.lumberjack.saplingsPlanted.1000=Tree school.
+achievement.farmer.carrotsHarvested.one=A Carrot!
+achievement.farmer.carrotsHarvested.25=Bugs bunny.
+achievement.farmer.carrotsHarvested.100=Carrots are good for your eyes, they said...
+achievement.farmer.carrotsHarvested.500=Sorry for the long post, here's a potato.
+achievement.farmer.carrotsHarvested.1000=Eh, what's up doc?
+achievement.farmer.potatoesHarvested.one=A Potato!
+achievement.farmer.potatoesHarvested.25=Potato salad.
+achievement.farmer.potatoesHarvested.100=Mr potato head.
+achievement.farmer.potatoesHarvested.500=Chips factory.
+achievement.farmer.potatoesHarvested.1000=Potatoes mister frodo.
+achievement.farmer.wheatHarvested.one=Wheat?
+achievement.farmer.wheatHarvested.25=Let's make some bread
+achievement.farmer.wheatHarvested.100=Smoke wheat everyday!
+achievement.farmer.wheatHarvested.500=Whole grain flower? No thank you!
+achievement.farmer.wheatHarvested.1000=Breadbasket.
+achievement.builder.hutsBuilt.one.desc=Your builder builds one hut.
+achievement.builder.hutsBuilt.25.desc=Your builder builds 25 huts.
+achievement.builder.hutsBuilt.100.desc=Your builder builds 100 huts.
+achievement.builder.hutsBuilt.500.desc=Your builder builds 500 huts.
+achievement.builder.hutsBuilt.1000.desc=Your builder builds 1000 huts.
+achievement.fisher.fishCaught.one.desc=Your fisherman catches one fish.
+achievement.fisher.fishCaught.25.desc=Your fisherman catches 25 fish.
```

Comment:
Wrong verb form.
It should be "Your builder built x huts"
"Your fisherman caught 25 fish"

I'd also say we should say "your fishermen" because it will be over all of them, no?

And in the case of the workers where the phrase is 100% the same but amount. I would make the amount a %d to make it only 1 language string.

Did you add them to poEditor?